locales/ruRU.lua

```diff
@@ -514,6 +514,28 @@ local L = app.L;
 		L.PROFILE_DELETE_TOOLTIP = "Удалить Выделенный Профиль";
 		L.PROFILE_SWITCH_TOOLTIP = "Установить Выделенный Профиль как Текущий Профиль\n\nИли кликните на Профиль с зажатым Shift";
 
+	-- Sync tab
+		L.SYNC = "Sync";
```

Comment:
Is this correct? 🤔
Didn't you want this instead?
```suggestion
		L.SYNC = "Синхронизация";
```